LVT/LVT.Services/EpicPresenter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using LVT.LVT.Interfaces;
+using System;
+using System.Collections.Generic;
+
+namespace LVT.LVT.Services
+{
+    public class EpicPresenter : IVisualizer<Epic> 
+
+    {
+        protected List<string[]> epicRowData = new List<string[]>();
+        
+
+        public string VisualizeToString(Epic epic, string parentNode)
+        {
+            return "[{ v: '" + epic.NodeID + "', f: 'Epic" + "<div style=\"font-style:italic\">" + $"{epic.Description}" + "</div>" +
+                                                                                   "<div style=\"font-style:italic\">" + $"{epic.Deadline}" + "</div>" +
+                                                                                   "'}, " + $"'{parentNode}']";
+        }
+
+        // this is not used
```

Comment:
If it's not used why is it in the code?

Reply:
oversight - removed now!

Reply:
This was a remnant of what I was initially trying to do - build lists with the relevant text snipppets inside and concatenate later (for every presenter class).  I think there's still a version of that around in a commit from 10 December (afternoon-ish) - the methods were called "ConvertToList".  This would have been closer to what Kieron was talking about earlier (I think).